packages/replay/src/integration.ts

```diff
@@ -48,15 +48,16 @@ export class Replay implements Integration {
     sessionSampleRate,
     errorSampleRate,
     maskAllText,
-    maskTextSelector,
     maskAllInputs = true,
+    maskTextClass = 'sentry-mask',
+    maskTextSelector,
     blockAllMedia = true,
-    _experiments = {},
     blockClass = 'sentry-block',
-    ignoreClass = 'sentry-ignore',
-    maskTextClass = 'sentry-mask',
     blockSelector = '[data-sentry-block]',
-    ...recordingOptions
+    ignoreClass = 'sentry-ignore',
+    _experiments = {},
+    recordingOptions = {},
```

Comment:
I would prefer if we pulled out the rrweb options that we want to allow. I still think having `recordingOptions` is a bit arbitrary when we have options like the above that could be considered recording options as well.

Reply:
Yeah, that's basically what we are doing here anyhow :) All the options we really support are on the top level, and `recordingOptions` is really more of an escape hatch to allow passing arbitrary other options (for which we do not provide any guarantees). That was also the original idea for calling this option `rrweb`, as that is really rrweb-specific stuff that we do not explicitly support.

Reply:
Yeah my suggestion is to kill the escape hatch is we are going to change the API anyway. I'm gonna go through the rrweb options and make a list of what I think we should support. Thoughts on that?

Reply:
I'm also generally fine with that, although we may get into the situation where one customer wants to do some kind of weird thing that rrweb supports but we do not want to support. But I'm also happy with just saying "no, sorry, can't do that" in such a case!